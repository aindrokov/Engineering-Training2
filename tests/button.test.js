import renderer from 'react-test-renderer';
import Button from '../src/components/button';

it('loads data when clicked', () => {
  const component = renderer.create(
    <Button/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});