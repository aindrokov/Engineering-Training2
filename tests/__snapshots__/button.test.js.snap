// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loads data when clicked 1`] = `
<button
  onClick={[Function]}
>
  Load Data
</button>
`;
